### PR TITLE
Initial draft for 2024 charter renewal

### PR DESCRIPTION
**Pull request by Yonet**
@himorin we will discuss this draft further, feel free to merge or leave.
 Thanks

### CODE REVIEW

**Comment by AdaRoseCannon** [comment]
Thank you Aysegul!

**Comment by AdaRoseCannon** [comment]
I don't seem able to merge on here but 👍

**Comment by Yonet** [comment]
@himorin can you merge please so we can look at it at the meeting. Thanks

**Comment by himorin** [comment]
merging for discussion at IWWG call

File: 2024/immersive-web-wg.html [hunk 1]
```diff
@@ -130,7 +130,7 @@ <h1 id="title">Immersive Web Working Group Charter</h1>
             Chairs
           </th>
           <td>
-            Ada Rose Cannon (Samsung), Chris Wilson (Google), Ayşegül Yönet (Microsoft)
+            Ada Rose Cannon (Samsung), Chris Wilson (Google), Ayşegül Yönet (ARtistus)
```
Comment (himorin) [inline 2024/immersive-web-wg.html:133]: this should be invited expert.

**Pull request merged**